applications/machine_learning/configuration/thingy53_nrf5340_cpuapp_ns/app_ZDebug.conf

```diff
@@ -0,0 +1,191 @@
+#
```

Comment:
Matter weather station is missing. We could add it or contact maintainers regarding this one.

Reply:
I already discussed that and they don't plan to add ns support so i left this.

Reply:
To be honest we don't have to do it either. The goal was to make sure it's possible.
I would still add the configuration to machine learning or some samples so we are sure this thing keeps building.